components/src/MonthRange/MonthRange.js

```diff
@@ -22,7 +22,8 @@ const MonthRange = ({
   labelProps,
   minDate,
   maxDate,
-  locale
+  locale,
+  disableAutoComplete
```

Comment:
i think this should be cased `disableAutocomplete` to stay consistent with `autocomplete` in Select